_posts/2015-11-30-translation-process.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: post
+title: Handling translations in a multi-platforms context
+permalink: /handling-translations/
+tags: translation,process
+category:
+author_id: adrienDeotto
+---
+
+Whether your Product is available on one platform or several, the question of how to handle translations is never easy. The reason of this difficulty is simple: translations require a very good coordination between different actors, at different stages of the Product development. People often tend to underestimate translations while they should actually be considered like proper features.
+
+In this article I will not focus on why it is important to have good copies in your product to drive engagement, reflect your product’s quality and define your brand’s positioning, instead, I will talk about the tools and processes which helped us handle translations at Onefootball.
+
+
+
+## The actors - who is involved in the translation process?
+
+At Onefootball, the following actors are involved, successively:
+
+* **Feature owner** – he/she is the responsible of a new feature and its definition, new copies included.
+* **Native speaker** – we always have a native English speaker reviewing a new feature’s copies to sign them off. Ideally this person is part of the Marketing team and can also align the tone of voice of the final copies. 
+* **Translation manager** – in our set-up, this role is covered by Technical Product Managers. They are responsible for preparing the ground for translation requests (we’ll see that later) and for co-ordinating the translators. At last, they are also responsible for making sure translations collected are propagated correctly to all platforms.
+* **Translators** –they can either be internal or external resources to the company. The important is that they are native speakers.
```

Comment:
Missing space before "they can" and I would also add a substantive after "important".